.gitignore

```diff
@@ -10,7 +10,7 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
-
+package-lock.json
```

Comment:
Good job, but based on NPM documentation It is highly recommended you commit the generated package-lock.json file.
read this: https://docs.npmjs.com/configuring-npm/package-locks.html